src/renderer/components/+workloads-deployments/deployments.tsx

```diff
@@ -86,14 +89,14 @@ export class Deployments extends React.Component<Props> {
           this.renderConditions(deployment),
         ]}
         renderItemMenu={(item: Deployment) => {
-          return <DeploymentMenu object={item}/>
+          return <DeploymentRestartMenu object={item}/>
         }}
       />
     )
   }
 }
 
-export function DeploymentMenu(props: KubeObjectMenuProps<Deployment>) {
+export function DeploymentScaleMenu(props: KubeObjectMenuProps<Deployment>) {
```

Comment:
Its still should be called `DeploymentMenu` since it must include different kind of menu items. So there must be only 1 menu function with restart and scale options in it. 

Can you please leave `DeploymentMenu` but expand it and include both restart and scale menu items inside of it? This way we will keep only 1 `kubeObjectMenuRegistry.add()` call and bring all items in one place.

Reply:
Fixed!